zilencer/management/commands/populate_db.py

```diff
@@ -181,15 +181,18 @@ def handle(self, **options: Any) -> None:
             zulip_realm_bots.extend(all_realm_bots)
             create_users(zulip_realm, zulip_realm_bots, bot_type=UserProfile.DEFAULT_BOT)
 
+            aaron = get_user("AARON@zulip.com", zulip_realm)
             zulip_webhook_bots = [
                 ("Zulip Webhook Bot", "webhook-bot@zulip.com"),
             ]
+            # If a stream is not supplied in the webhook URL, the webhook
+            # will (in some cases) send the notification as a PM to the
+            # owner of the webhook bot, so bot_owner can't be None
             create_users(zulip_realm, zulip_webhook_bots,
-                         bot_type=UserProfile.INCOMING_WEBHOOK_BOT)
+                         bot_type=UserProfile.INCOMING_WEBHOOK_BOT, bot_owner=aaron)
```

Comment:
I'd prefer another user be the owner; "AARON" is primarily there for capitalization testing :)

Reply:
Did you do an audit for whether there's other ways a bot owner could be None, BTW?

Reply:
I did a check for this in our production environments, and the old bots without an owner were system bots (which aren't webhooks).  So I think we're OK, but maybe worth a code read.

Reply:
@timabbott: I already did an audit. See [this backend](https://github.com/zulip/zulip/blob/master/zerver/views/users.py#L332), which is called by the UI interface when a new bot is created. Unless I am mistaken, `bot_owner` shouldn't ever be `None`. :)

Reply:
Great, sgtm :)